### ISSUE TITLE
Add Publications to the Website

### ISSUE THREAD

**Issue by bonaventuredossou** [blackatmila/blackatmila.github.io#2]
[Bonaventure F. P. Dossou](https://bonaventuredossou.github.io/)

- Longitudinal wearable physiological data encode digital biomarkers that enable birth timing prediction with deep learning models: A Retrospective analysis from 3000+ Fitbit Users: Mercy Asiedu, Nichole Young-Lin, **Bonaventure F. P. Dossou**, Maja Matarić, Conor Heneghan, Katherine Heller, Belen Lafon (under review in Nature Medicine) 
- Active Learning with Non-Uniform Costs for African Natural Language Processing: **Bonaventure F. P. Dossou**, Ines Arous, Audrey Durand, Jackie Cheung (EACL 2026)  
- Towards Open-Ended Discovery for Low-Resource NLP: **Bonaventure F. P. Dossou** and Henri Aïdasso (EMNLP 2025) 
- Early Prediction of Postpartum Mood Disorders from Longitudinal Wearable Biometrics using deep learning and time series generative adversarial networks: **Bonaventure F. P. Dossou**, Mercy Nyamewaa Asiedu, Maja Mataric, Katherine A. Heller, Belen Lafon, Nichole Young-Lin (MLHC 2025, under preparation for Nature Medicine)  
- Adapting Pretrained ASR Models to Low-resource Clinical Speech using Epistemic Uncertainty-based Data Selection: **Bonaventure F. P. Dossou** (ACL 2025) 
- Rethinking Full Finetuning from Pretraining Checkpoints in Active Learning for African Languages: **Bonaventure F. P. Dossou**, Ines Arous, Jackie Cheung (ACL 2025) 
- CodeUnlearn: Amortized Zero-Shot Machine Unlearning in Language Models Using Discrete Concept: YuXuan Wu, **Bonaventure F. P. Dossou**, Dianbo Liu (arXiv preprint)  
- InkubaLM: A small language model for low-resource African languages: Atnafu Lambebo Tonja*, **Bonaventure F. P. Dossou***, Jessica Ojo, Jenalea Rajab, Fadel Thior, Eric Peter Wairagala, Aremu Anuoluwapo, Pelonomi Moiloa, Jade Abbott, Vukosi Marivate, Benjamin Rosman (arXiv preprint)  
- AfriMed-QA: Towards A Pan-African, Multi-Specialty, Medical Question-Answering Benchmark Dataset: Tobi Olatunji, Charles Nimo, ..., **Bonaventure F. P. Dossou**, ... (ACL 2025)  
- FonMTL: Towards Multitask Learning for the Fon Language: **Bonaventure F. P. Dossou**, Iffanice Houndayi, Pamely Zantou, and Gilles Hacheme (EMNLP 2023)  
- GFlowOut: Dropout with Generative Flow Networks: Dianbo Liu, Moksh Jain, **Bonaventure F. P. Dossou**, Qianli Shen, Salem Lahlou, Anirudh Goyal, Nikolay Malkin, Chris Emezue, Dinghuai Zhang, Nadhir Hassen, Xu Ji, Kenji Kawaguchi, Yoshua Bengio (ICML 2023) 
- Pretrained Vision Models for Predicting High-Risk Breast Cancer Stage: **Bonaventure F. P. Dossou**, Yeno Gbenou, Miglanche Ghomsi (ICLR 2023)  
- AfriNames: Most ASR models “butcher” African Names: Tobi Olatunji, Tejumade Afonja, **Bonaventure F. P. Dossou**, Atnafu Lambebo Tonja, Chris Chinenye Emezue, Amina Mardiyyah Rufai, Sahib Singh (Interspeech 2023)  
- Self-Active Learning for Multilingual Language Models: Case Study of 23 African Languages: **Bonaventure F. P. Dossou**, Atnafu Lambebo Tonja, Oreen Yousuf, Salomey Osei, Abigail Oppong, Iyanuoluwa Adeola Shode, and Oluwabusayo Olufunke Awoyomi (EMNLP 2022)  
- Biological Sequence Design with GFlowNets: Moksh Jain, Emmanuel Bengio, Alex-Hernandez Garcia, Jarrid Rector-Brooks, **Bonaventure F. P. Dossou**, ..., Payel Das, Yoshua Bengio (ICML 2022)  
- GraphCC for Diverse and Novel Antimicrobial Peptides Generation and Selection: **Bonaventure F. P. Dossou**, Dianbo Liu, Xu Ji, Moksh Jain, Almer M. van der Sloot, Roger Palou, Michael Tyers, Yoshua Bengio (arxiv preprint)
- MeSH2Matrix: Machine learning-driven biomedical relation classification based on the MeSH keywords of PubMed scholarly publications: Houcemeddine Turki, **Bonaventure F. P. Dossou**, …, Afif Masmoudi (ECIR 2022)  
- MMTAfrica: Multilingual Machine Translation for African Languages: Chris C. Emezue and **Bonaventure F. P. Dossou** (EMNLP 2021)  
- FSER: Deep Convolutional Neural Networks for Speech Emotion Recognition: **Bonaventure F. P. Dossou** and Yeno Gbenou (ICCV 2021)  
- OkwuGbé: End-to-End Speech Recognition for Fon and Igbo: **Bonaventure F. P. Dossou** and Chris Emezue (EMNLP 2021)  
- AfriVec: Word Embedding Models for African Languages. Case Study of Fon and Nobiin: **Bonaventure F. P. Dossou** and Mohammed Sabry (EACL 2021)  
- Crowdsourced Phrase-Based Tokenization for Low-Resourced Neural Machine Translation: The case of Fon Language: **Bonaventure F. P. Dossou** and Chris C. Emezue (EACL 2021)  
- FFR: Fon-French Neural Machine Translation: **Bonaventure F. P. Dossou** and Chris C. Emezue (ACL 2020)
